### PR TITLE
Use redux state inside components

### PR DESCRIPTION
**Pull request by DmitriyKrivovyaz**

### CODE REVIEW

File: client/src/layouts/cityForecast/CurrentCityForecast/CurrentCityForecast.jsx [hunk 1]
```diff
@@ -2,17 +2,17 @@ import React from 'react';
...
   return (
     <div className={classes.container}>
       <img className={classes.weatherIcon} src={moonWeatherIcon} alt="weahterIcon" />
       <div className={classes.weather}>
-        <span className={`${classes.city} ${classes.weatherItem}`}>{currentCityData.city}</span>
+        <span className={`${classes.city} ${classes.weatherItem}`}>{location.name}</span>
         <span
           className={`${classes.temperature} ${classes.weatherItem}`}
-        >{`${currentCityData.temperature} °C`}</span>
+        >{`${weather.temperature} °C`}</span>
```
Comment (ximet) [inline client/src/layouts/cityForecast/CurrentCityForecast/CurrentCityForecast.jsx]: move °C to const.

**Review by ximet — APPROVED** [review]

File: client/src/layouts/hourlyForecasts/Container.jsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { connect } from 'react-redux';
+import HourlyForecasts from './HourlyForecasts';
+
+const mapStateToProps = state => ({
+    hourlyForecasts: state.hourlyWeather.hourlyWeather
```
Comment (anton-vlasik) [inline client/src/layouts/hourlyForecasts/Container.jsx]: I'm seeing a pattern with `state.thing.thing` in few places. Could we eliminate one of those `thing`s?

File: client/src/layouts/hourlyForecasts/HourlyForecasts.jsx [hunk 3]
```diff
@@ -2,19 +2,20 @@ import React from 'react';
 import Slider from '../../components/slider/Slider';
 import classes from './hourlyForecasts.module.css';
 import HourlyForecast from './hourlyForecast/HourlyForecast';
-import { hourlyForecasts } from '../../mock/mock';
+// import { hourlyForecasts } from '../../mock/mock';
```
Comment (anton-vlasik) [inline client/src/layouts/hourlyForecasts/HourlyForecasts.jsx]: remove commented code

File: client/src/layouts/hourlyForecasts/hourlyForecast/HourlyForecast.jsx [hunk 4]
```diff
@@ -3,22 +3,25 @@ import classes from './hourlyForecast.module.css';
...
+  const time = formatDate(forecast.time).time
+
   return (
     <div className={classes.container}>
-      <span className={classes.time}>{forecast.time}</span>
+      <span className={classes.time}>{time}</span>
       <span className={classes.temperature}>{`${forecast.temperature}°C`}</span>
       <div>
-        <img className={classes.weatherIcon} src={forecast.icon} alt="weather-icon" />
+        <img className={classes.weatherIcon} src={moonAndCloudsIcon} alt="weather-icon" />
```
Comment (anton-vlasik) [inline client/src/layouts/hourlyForecasts/hourlyForecast/HourlyForecast.jsx]: why this icon is exactly `moonAndClouds`?

Comment (DmitriyKrivovyaz) [reply client/src/layouts/hourlyForecasts/hourlyForecast/HourlyForecast.jsx]: i will change it later in next pr, but here it was out of scope of this pr

**Review by anton-vlasik — APPROVED** [review]

**Pull request merged**